packages/ovh-manager/server/proxy/index.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  Aapi: require('./Aapi'), // eslint-disable-line global-require
```

Comment:
noooooooooooooo ! :)

You should do the `require` at the top of file, and use the created var instead :)